kura/distrib/src/main/resources/common/comment-interfaces-file.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import subprocess
+import sys
+import shutil
+import logging
+from os.path import exists
+
+ETHERNET_TYPES = ['ethernet', 'eth', 'wired']
+WIRELESS_TYPES = ['wifi', 'wireless']
+INTERFACES_PATH = "/etc/network/interfaces"
+INTERFACES_OLD_PATH = INTERFACES_PATH + ".old"
+INTERFACES_TMP_PATH = INTERFACES_PATH + ".tmp"
+
+def get_eth_wlan_interfaces_names():
```

Comment:
Instead of duplicating the code here, you can import the function from `find-net-interfaces` and proceed from there.